sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionStateBuilder.scala

```diff
@@ -101,6 +102,7 @@ class HiveSessionStateBuilder(session: SparkSession, parentState: Option[Session
       override def strategies: Seq[Strategy] = {
         experimentalMethods.extraStrategies ++
           extraPlanningStrategies ++ Seq(
```

Comment:
can we change it to `super. strategies ++ Seq(HiveTableScans, Scripts)`?

Reply:
Sure, I will change it.

Reply:
Looks like the ordering matters, If I put Hive related strategies in the end, some unit tests will be failed.